lib/APPRemoteControl/StartupState.h

```diff
@@ -89,17 +89,30 @@ class StartupState : public IState
      */
     void exit() final;
 
+    /**
+     * Notify the state, that the initial data set was received.
+     */
+    void notifyInitialDataIsSet()
+    {
+        m_initialDataSet = true;
+    }
+
 protected:
 private:
     /**
      * Duration in ms how long the application name shall be shown at startup.
      */
-    static const uint32_t APP_NAME_DURATION = 2000;
+    static const uint32_t APP_NAME_DURATION = 2000U;

```

Comment:
a good habbit is to tell in what physical unit the value is i.e.
APP_NAME_DURATION**_MS**  = 2000U;